traitsui/qt4/image_editor.py

```diff
@@ -66,13 +66,13 @@ def paintEvent(self, event):
         """
         pixmap = self._pixmap
         if pixmap is None:
-            return
+            super().paintEvent(event)
 
         pm_size = pixmap.size()
         pm_width = pm_size.width()
         pm_height = pm_size.height()
         if pm_width == 0 or pm_height == 0:
-            return
+            super().paintEvent(event)
```

Comment:
This seems ok, but why do we need it exactly?

Reply:
This delegates to the base class to do the painting - that seems reasonable when we don't have an image to render.